task/notify-matrix/0.1/README.md

```diff
@@ -0,0 +1,35 @@
+# `notify-matrix`
+
+The `notify-marix` `Task` sends messages to a [Matrix](https://matrix.org/) room on its corresponding endpoint.
+
+## Parameters 
+
+This `Task` has the following required inputs:
+
+1. **`matrix-secret`**: the name of a secret, containing a valid matrix access token (see below)
+2. **`room`**: the matrix roomID where the notification will be sent, in the format `#ROOM_NAME:SERVER_NAME`
+3. **`endpoint`**: URI of the matrix server to connect and send the message from
+4. **`message`**: the message to be sent
+
+## Setting up the `matrix-secret`
```

Comment:
It would be cool to have a recipe on how to retrieve an access token - e.g. an example with a curl to execute in local shell in order to have a secret that could be added into `kind: Secret`

Reply:
sure! I'll update the Readme